### PR TITLE
Add database

### PR DESCRIPTION
**Pull request by Thehashhobo**
small merge to add a comment database.

### CODE REVIEW

**Review by reganq — APPROVED** [review]
This looks good to me

File: src/main/java/Database/comments.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"commentID","commenterID","body","creationDate"
+"0","1","this is comment 0 made by user 1","2022-11-20"
```
Comment (JohannLeidenfrost) [inline src/main/java/Database/comments.csv:2]: Please remove the test data here!

**Review by JohannLeidenfrost — COMMENTED** [review]
Looks good except please remove the testData here!

File: src/main/java/Database/comments.csv (hunk 1 above)
Comment (Thehashhobo) [reply src/main/java/Database/comments.csv:2]: ok

**Pull request closed without merging**